src/utils/columns/getColumnSortDirectionIcon.ts

```diff
@@ -0,0 +1,16 @@
+import { Sort } from "@dashboard/types";
+
+export function getColumnSortDirectionIcon<T extends string = string>(
+  { sort, asc }: Sort<T>,
+  columnName: T,
+) {
```

Comment:
**nitpick**: return type would be nice here, this way we can easily determine if that function is reusable in a given case